crates/chronicle/src/bootstrap/mod.rs

```diff
@@ -212,22 +229,15 @@ pub async fn graphql_server<Query, Mutation>(
     options: &ArgMatches,
     jwks_uri: Option<Url>,
     id_pointer: Option<String>,
-    opa_executor: WasmtimeOpaExecutor,
+    opa: Arc<Mutex<WasmtimeOpaExecutor>>,
```

Comment:
This is fine for now, but we may want to look at moving this to process in spawn_blocking depending on how our profiling turns out